### PR TITLE
Add handler for start endpoint.

### PR DESCRIPTION
**Pull request by mirekgab**

### CODE REVIEW

**Review by dawciobiel — DISMISSED** [review]
Looks good for me

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/TrackExpensesAppControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class TrackExpensesAppControllerTest {
+
+    @InjectMocks
+    TrackExpensesAppController trackExpensesAppController;
+
+    @Test
+    void sayHello() {
+        String result = trackExpensesAppController.sayHello();
+        assertTrue(result.startsWith("TrackExpensesApp by"));
+    }
```
Comment (dawciobiel) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/TrackExpensesAppControllerTest.java:20]: There should be one empty line between 20 and 21.

**Review by dawciobiel — APPROVED** [review]
Looks good for me

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/TrackExpensesAppControllerTest.java (hunk 1 above)
Comment (mirekgab) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/TrackExpensesAppControllerTest.java:20]: Done.

**Review by dawciobiel — APPROVED** [review]
As usual looks good for me

**Review by odzio33 — APPROVED** [review]
Looks good for me

**Pull request merged**